build/makelib/golang.mk

```
@@ -166,38 +163,20 @@ go.fmt: $(GOFMT)
 
 go.validate: go.vet go.fmt
 
-.PHONY: go.vendor.lite
-go.vendor.lite: $(DEP)
-#	dep ensure blindly updates the whole vendor tree causing everything to be rebuilt. This workaround
-#	will only call dep ensure if the .lock file changes or if the vendor dir is non-existent.
-	@if [ ! -d $(GO_VENDOR_DIR) ]; then \
-		$(MAKE) go.vendor; \
-	elif ! $(DEP) ensure -no-vendor -dry-run &> /dev/null; then \
-		$(MAKE) go.vendor; \
-	fi
+.PHONY: go.mod
+go.mod:
+	@echo === ensuring modules are tidied
+	@$(GOHOST) mod tidy
+
+.PHONY: go.mod.update
+go.mod.update:
+	@echo === updating modules
+	@$(GOHOST) get -u ./...
 
 .PHONY: go.vendor.check
-go.vendor.check: $(DEP)
-	@echo === checking if vendor deps changed
-	@$(DEP) check -skip-vendor
-	@echo === vendor deps have not changed
-
-.PHONY: go.vendor
-go.vendor: $(DEP)
-	@echo === ensuring vendor dependencies are up to date
-	@$(DEP) ensure
-
-.PHONY: go.vendor.update
-go.vendor.update: $(DEP)
-	@echo === updating vendor dependencies
-	@$(DEP) ensure -update -v
-
-$(DEP):
-	@echo === installing dep
-	@mkdir -p $(TOOLS_HOST_DIR)/tmp
-	@curl -sL -o $(DEP) https://github.com/golang/dep/releases/download/$(DEP_VERSION)/dep-$(GOHOSTOS)-$(GOHOSTARCH)
-	@chmod +x $(DEP)
-	@rm -fr $(TOOLS_HOST_DIR)/tmp
+go.vendor.check:
```

```suggestion
go.mod.check:
```

Shouldn't this be like this? ^

---

I fixed it. Thanks.